src/hostname/hostnamectl.c

```diff
@@ -52,6 +53,7 @@ typedef struct StatusInfo {
         const char *hardware_vendor;
         const char *hardware_model;
         const char *firmware_version;
+        const usec_t firmware_date;
```

Comment:
drop the "const" btw. that only is relevant for pointers, and useless otherwise.